docs/predict-api/v2/required_api.md

```diff
@@ -733,64 +769,64 @@ one-dimensional, row-major order of the tensor elements. Element
 values must be given in "linear" order without any stride or padding
 between elements.
 
+Using a "raw" representation of tensors with
+ModelInferRequest::raw_input_contents and
+ModelInferResponse::raw_output_contents will typically allow higher
+performance due to the way protobuf allocation and reuse interacts
+with GRPC. For example, see https://github.com/grpc/grpc/issues/23231.
+
+An alternative to the "raw" representation is to use
+InferTensorContents to represent the tensor data in a format that
+matches the tensor's data type.
+
     //
-    // The data contained in a tensor. For a given data type the
-    // tensor contents can be represented in "raw" bytes form or in
-    // the repeated type that matches the tensor's data type. Protobuf
-    // oneof is not used because oneofs cannot contain repeated fields.
+    // The data contained in a tensor represented by the repeated type
+    // that matches the tensor's data type. Protobuf oneof is not used
+    // because oneofs cannot contain repeated fields.
     //
     message InferTensorContents
     {
-      // Raw representation of the tensor contents. The size of this
-      // content must match what is expected by the tensor's shape
-      // and data type. The raw data must be the flattened, one-dimensional,
-      // row-major order of the tensor elements without any stride or padding
-      // between the elements. Note that the FP16 data type must be
-      // represented as raw content as there is no standard support for a
-      // 16-bit float type.
-      bytes raw_contents = 1;
-
       // Representation for BOOL data type. The size must match what is
       // expected by the tensor's shape. The contents must be the flattened,
       // one-dimensional, row-major order of the tensor elements.
-      repeated bool bool_contents = 2;
+      repeated bool bool_contents = 1;
 
```

Comment:
So this is just proposal and not implemented by anyone right? I'm not familiar with this part of work, just think we should not change the number field once finalized to avoid the compatibility issues.

Reply:
Triton implements it as a beta so small changes like this are ok for it.

Reply:
Got it, thanks.